scripts/ci-entrypoint.sh

```diff
@@ -30,6 +30,7 @@ cd "${REPO_ROOT}" && make "${KUBECTL##*/}"; make "${HELM##*/}"
 # export the variables so they are available in bash -c wait_for_nodes below
 export KUBECTL
 export HELM
+export KUBERNETES_VERSION="${KUBERNETES_VERSION:-}"
```

Comment:
do we need to do this or can we just replace `${KUBERNETES_VERSION}` with `${KUBERNETES_VERSION:-}` below?

Reply:
This is probably more resilient. I think it's correct that as long as we refer to `${KUBERNETES_VERSION:-}` the first time we use it we are O.K. (no unbound variable error), but the next maintainer of this file that introduces a novel `${KUBERNETES_VERSION}` may not follow that pattern and it may become the _new_ initial reference.

Reply:
That's fair. The other variables are all defaulting for every use below but this works too. Let's maybe move it down to where the variable is used though, this section is specific to the binary tools and the comment on line 30 doesn't apply.

Reply:
I've gone ahead and does the inline default assignment thing as that's the existing pattern. I think we can trust our CI to complain if/when a future change breaks this.